src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -117,11 +118,19 @@ public function generateAblySignature($channelName, $socketId, $userData = null)
      * @param  string  $event
      * @param  array  $payload
      * @return void
+     *
+     * @throws \Illuminate\Broadcasting\BroadcastException
      */
     public function broadcast(array $channels, $event, array $payload = [])
     {
-        foreach ($this->formatChannels($channels) as $channel) {
-            $this->ably->channels->get($channel)->publish($event, $payload);
+        try {
+            foreach ($this->formatChannels($channels) as $channel) {
+                $this->ably->channels->get($channel)->publish($event, $payload);
+            }
+        } catch (AblyException $e) {
+            throw new BrodcastException(
```

Comment:
This exception is misspelt and missing an import

Reply:
Thanks